Lib/test/_test_multiprocessing.py

```diff
@@ -269,6 +269,62 @@ def _test(cls, q, *args, **kwds):
             q.put(bytes(current.authkey))
             q.put(current.pid)
 
+    def test_parent_process_attributes(self):
+        if self.TYPE == "threads":
+            self.skipTest('test not appropriate for {}'.format(self.TYPE))
+        from multiprocessing.process import current_process
+        rconn, wconn = self.Pipe(duplex=False)
+        p = self.Process(target=self._test_send_parent_process, args=(wconn,))
+        p.start()
+        p.join()
+        parent_process = rconn.recv()
+        assert parent_process.pid == current_process().pid
```

Comment:
Could you also check that it is equal to `os.getpid()`? This way, it check that there is no border effects on the `current_process`.